examples/device/audio_test/src/tusb_config.h

```diff
@@ -91,19 +91,21 @@ extern "C" {
 // AUDIO CLASS DRIVER CONFIGURATION
 //--------------------------------------------------------------------
 
+#ifndef AUDIO_SAMPLE_RATE
+#define AUDIO_SAMPLE_RATE                                             48000
```

Comment:
this should be moved to main.c . `tusb_config.h` is for stack configurable value. In practical, application can use it as project config, although since we are making example for usb configure. It is best to make it explicit.

Reply:
Ok fine! The same is done within the `uac2_headset` example, which is why i did that in the simple example too! You may change it there as well! :)

Reply:
Yup, already done that with headset example with latest commit